pyproject.toml

```
@@ -31,7 +31,7 @@ description = "A game based on Pydeew Valley by ClearCode, used by the Universit
 # Use Ruff to replace Black, isort, and Flake8
 [tool.ruff]
 line-length = 88
-target-version = "py312"
+target-version = "py38"
```

```suggestion
target-version = "py312"
```

Why did you modify this? This project is using Python 3.12. You fix linting and formatting errors by running the scripts and making changes, not changing the settings.